README.md

```diff
@@ -47,3 +47,10 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+## Answer
+
+If you run this algorithm with $n$ elements, it will run the heldKarp() function $n$ times.  Then, each of those will recursively run it $n - 1$ times, and so on until it hits the base case.  However, if the function has already been calculated and stored in the cache, it doesn't have to evaluate it again.  Thus, the algorithm will run for every subset of the n items, which is $2^n$ times.  Each time it runs the function, it copies the cities array in newCities ($n$ time), and then runs a splice function, which could also take $n$ time.  This gives us a time complexity of $n + n + 2^n$, which simplifies to $\Theta(2^n)$.
```

Comment:
So $n$ time happens for each of the $2^n$ subsets? How would you combine those runtimes?